blueman/gui/manager/ManagerDeviceList.py

```diff
@@ -85,7 +85,7 @@ def __init__(self, adapter=None, inst=None):
 
         self.set_search_equal_func(self.search_func, None)
 
-        self._faderhandler: Optional[int] = None
+        self._faderhandlers = Dict[str, int]
```

Comment:
The syntax is wrong here. `self._faderhandlers: Dict[str, int] = {}` should be what you want. Otherwise you're assigning the type.

Reply:
@cschramm Thanks. Fixed.